lecsum-fe/pages/wrong_answer.py

```
@@ -3,170 +3,174 @@
-                on_change=toggle_selection,
-                args=(quiz_id,)
+    # 세션 스테이트 초기화
+    if 'selected_quiz_ids' not in st.session_state:
+        st.session_state['selected_quiz_ids'] = []
+
+    # API에서 틀린 문제 조회
+    @st.cache_data(ttl=60)
+    def fetch_wrong_answers(limit=50, offset=0):
+        """오답노트 API 호출"""
+        try:
+            response = requests.get(
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)